app/views/articles/_sidebar.html.erb

```diff
@@ -41,7 +41,7 @@
         <div class="side-footer">
           <a href="/about">About</a>
           <a href="/sponsors">Sponsors</a>
-          <a href="https://shop.dev.to/"><%= ApplicationConfig["COMMUNITY_NAME"] %> Shop</a>
+          <a href="<%= SiteConfig.shop_url %>"><%= community_name %> Shop</a>
```

Comment:
I wuould put a "if shopurl present" here, as a community might not have the shop at all and thus this link should not be rendered